overlord/ifacestate/ifacemgr.go

```diff
@@ -174,10 +174,6 @@ func (m *InterfaceManager) StartUp() error {
 			return err
 		}
 	}
-	if snapdAppArmorServiceIsDisabled() {
```

Comment:
Otherwise would it help if we spawned this in a separate go thread so it'd run the check in the background but didn't block? I don't know how important this warning is but it seems nice to have. Maybe @mvo5 has an opinion on this?

Reply:
Yeah, delaying seems slightly better, it could be a go routine with a 5s initial wait or something to not add more stress to systemd. I looked at the history and the change fixes a real bug (https://bugs.launchpad.net/snapd/+bug/1806135) and iirc/afaik some systems like arch need to always manually enable apparmor too

Reply:
Could we instead add `ConditionSecurity=apparmor` in `snapd.apparmor.service`. Then add `Wants=snapd.apparmor.service` in `snapd.service`?

Reply:
If have tweaked the systemd units so that this issue will not happen again.

Reply:
Aha, nice! So this ensures that when snapd.service is enabled snapd.apparmor gets enabled by default too?

Reply:
Technically, it is not "enabled". But if snapd is started, it will pull snapd.apparmor along.

There are also ways to "enable" other units with `Also=` in the `[Install]` section. But this is not what we need here in my opinion. This would be useful, if you want to enable snapd.apparmor, but not snapd. Which I do not think is a valid use case.

Reply:
"if you want [to be able] to enable"